docs/release-notes/deprecate-containerRuntime.rst

```diff
@@ -0,0 +1,10 @@
+:orphan:
+
+**Deprecations**
+
+-  AgentRM: Launching a Singluarity/Podman/Apptainer container runtimes for Agent is no longer
+   supported. Docker is the only option that is supported. This change only affects when
+   container_runtime is set to podman, using a podman emulation layer is unchanged. If you want to
+   use singularity, podman, or apptainer the Determined master enterprise edition still supports it.
```

Comment:
> Docker is the only option that is supported. This change only affects when
   container_runtime is set to podman, using a podman emulation layer is unchanged.
   
This is confusing to me.  Maybe something like this would be clearer: "Docker and Podman using the emulation layer are still supported."

> If you want to
   use singularity, podman, or apptainer the Determined master enterprise edition still supports it.

Is this statement accurate?  Are you able to run AgentRM with singularity, podman, or apptainer in EE?  If so, then wouldn't it be the same amount of work to support all containers in both EE and OSS?  Why bother deprecating it?

Reply:
> If you want to
use singularity, podman, or apptainer the Determined master enterprise edition still supports it.

This came from a comment from Nick: https://github.com/determined-ai/determined/pull/9516#discussion_r1662869681

I'm not exactly sure what he means. I _think_ he was referring to how the other container runtimes (podman/singularity/apptainer) still work with slurm (and slurm is ee only). If I recall correctly, podman also works with kubernetesrm though ... 

If it's accurate, I think I'd rather say something like "Kubernetes and Slurm resource managers still support singularity, podman, or apptainer use."